config/config_r.r

```diff
@@ -1,7 +1,10 @@
-Main <- function(){
-    CRAN_packages   <- c("yaml", "ggplot2")
-    GitHub_packages <- NULL
-    # GitHub packages should be the full path to the relevant repo
+# Required packages installed from CRAN.
+CRAN_packages   <- c("yaml")
+# Required packages installed from Github. Should be the full path to the relevant repo.
+GitHub_packages <- NULL
+
+main <- function(CRAN_packages = NULL, GitHub_packages = NULL) {
```

Comment:
We might want to add arguments to control the install_CRAN options for repo, dependency, and quiet. We'll set their defaults to the specs you've already filled in.

Reply:
Another question is, should we force users to re-install the packages if they already install the package before? We might want to make this consistent between python, r and stata.

Reply:
I'd vote no. We don't force a re-installation unless we need to specify a specific version as with gslab_python. I recognize that this creates an inconsistency between python/R and stata, but I'm fine with that, unless you know an easy way to check if a stata package has been installed.